staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -83,6 +83,10 @@ import (
 
 const kubectlCmdHeaders = "KUBECTL_COMMAND_HEADERS"
 
+var (
+	allowedCmdsSubcommandPlugin = map[string]struct{}{"create": {}}
```

Comment:
Nit: `sets.Set[string]` will be more appropriate and easier to work with by using `Has` etc.